notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>"to file" is redundant</p><p> </p><p>you can now -&gt; you now</p><p> </p><p>Storage -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>"To file": Ack (see previous).</p><p> </p><p>s/you can now/you now: Done.</p><p> </p><p>"Cloud Storage" is correct branding. See me if you have questions.</p>